src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User tmpUsr : users) {
```

Comment:
```suggestion
        for (User user : users) {
```
it is difficult to read shortened names, let's not use them